resources/ui/outputhelpwin.ui

```diff
@@ -34,7 +34,7 @@ p, li { white-space: pre-wrap; }
+&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${nsmclientdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - client directory within the NSM session directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${desktopdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user desktop directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${downloaddir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user download directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${documentsdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user documents directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${musicdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user music directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${picturesdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user pictures directory&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; font-size:12pt;&quot;&gt;${videosdir}&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; - localized XDG user video directory&lt;/span&gt;&lt;/p&gt;
```

Comment:
What exactly is the typo that was fixed here? It's very hard to see in this diff.

Reply:
_localized XDG user music directory_ -> _localized XDG user documents directory_